packages/react/src/primitives/Expander/Expander.tsx

```diff
@@ -1,24 +1,28 @@
 import classNames from 'classnames';
 import { Root } from '@radix-ui/react-accordion';
+import * as React from 'react';
 
-import { Primitive } from '../types/view';
+import { PrimitiveWithForwardRef } from '../types/view';
 import { ExpanderProps } from '../types/expander';
 import { ComponentClassNames } from '../shared/constants';
 import { splitPrimitiveProps } from '../shared/styleUtils';
 
-export const Expander: Primitive<ExpanderProps, typeof Root> = ({
-  children,
-  className,
-  defaultValue,
-  isCollapsible,
-  onChange,
-  // It is not in use but remove it from rest to avoid type errors
-  onValueChange,
-  testId,
-  type = 'single',
-  value,
-  ..._rest
-}) => {
+const ExpanderPrimitive: PrimitiveWithForwardRef<ExpanderProps, typeof Root> = (
+  {
+    children,
+    className,
+    defaultValue,
+    isCollapsible,
+    onChange,
+    // It is not in use but remove it from rest to avoid type errors
+    onValueChange,
```

Comment:
I can't remember why this is included. Is this coming from `typeof Root`? If so, can we use Omit to remove it if we aren't using it?

Reply:
Yes, it is coming from `typeof Root`. `Omit` doesn't work in this case, it only works for `interface` shallowly :(